doc/users/plugins.rst

```diff
@@ -82,6 +82,10 @@ Optional arguments::
   exceed the total amount of resources available (memory and threads), when
   ``False`` (default), only a warning will be issued.
 
+  maxtasksperchild : refresh the workers after this specific number of nodes
+  run (default: 10).
```

Comment:
Actually, this could be clearer. Perhaps: "number of nodes to run on each worker before refreshing (default: 10)."